### ISSUE TITLE
@lw macro in texinfo-inferred-body causes makeinfo error

### ISSUE THREAD

**Issue by rpgoldman** [nikodemus/sb-texinfo#8]
`texinfo-inferred-body` dumps a class precedence list as follows:
``` lisp
(format *texinfo-output* "Class precedence list: @code{~(~{@lw{~A}~^, ~}~)}~%~%"
              (remove-if (lambda (class)  (hide-superclass-p name class))
                         (mapcar #'class-name (ensure-class-precedence-list (find-class name)))))
```
Unfortunately, this causes an error in makeinfo, as follows:
```
include/struct-shopunif-binding.texinfo:4: warning: @iftex should only appear at the beginning of a line (possibly involving @lw)
include/struct-shopunif-binding.texinfo:4: warning: @iftex should only appear at the beginning of a line (possibly involving @lw)
include/struct-shopunif-binding.texinfo:4: superfluous argument to @end ifnottex: , structure-object@c (possibly involving @lw)
include/struct-shopunif-binding.texinfo:4: warning: @iftex should only appear at the beginning of a line (possibly involving @lw)
include/struct-shopunif-binding.texinfo:4: superfluous argument to @end ifnottex: , t@c (possibly involving @lw)
include/struct-shopunif-binding.texinfo:4: misplaced }
include/struct-shopunif-binding.texinfo:4: @code missing closing brace
include/struct-shopunif-binding.texinfo:4: warning: @iftex should only appear at the beginning of a line (possibly involving @lw)
include/struct-shopunif-binding.texinfo:4: warning: @iftex should only appear at the beginning of a line (possibly involving @lw)
include/struct-shopunif-binding.texinfo:4: superfluous argument to @end ifnottex: , structure-object@c (possibly involving @lw)
include/struct-shopunif-binding.texinfo:4: warning: @iftex should only appear at the beginning of a line (possibly involving @lw)
include/struct-shopunif-binding.texinfo:4: superfluous argument to @end ifnottex: , t@c (possibly involving @lw)
include/struct-shopunif-binding.texinfo:4: misplaced }
include/struct-shopunif-binding.texinfo:4: @code missing closing brace
include/struct-shopunif-binding.texinfo:2: @anchor `Structure shopunif:binding' previously defined
```
This happens because `@lw` is used in the middle of an output line, which violates this constraint on texinfo:
> Macros containing a command which must be on a line by itself, such as a conditional, cannot be invoked in the middle of a line. Similarly, macros containing line-oriented commands or text, such as @example environments, may behave unpredictably in TEX. (from the Texinfo manual for version 6.6)

Unfortunately, I am not sure what the `@lw` macro is supposed to do, or I would provide a patch.  Any suggestions?

**Comment by rpgoldman** [comment]
@nikodemus If you get a chance, applying this patch would be nice.  Or, if you would like to let me push, and you like this patch, I can do it myself.

**Comment by nikodemus** [comment]
I've added you as a collaborator :)

**Comment by rpgoldman** [comment]
Fixed in #10.
